.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Unit Tests
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - run: npm install
```

Comment:
I think npm ci is a better practice

Reply:
okay, will change in next commit 👍

Reply:
implemented in https://github.com/icgc-argo/argo-dictionary/pull/45/commits/7471c5bc8ebab986ba526477542298906fa35040